atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -88,19 +88,18 @@ static void waitForSchemaVersions(
         long sleepTime = INITIAL_SLEEP_TIME;
         Map<String, List<String>> versions;
         do {
-            // This only returns the schema versions of nodes that the client knows exist. In particular, if a node we
-            // shook hands with goes down, it will have schema version UNREACHABLE; however, if we never shook hands
-            // with a node, there will simply be no entry for it in the map. Hence the check for the number of nodes.
+            // This may only include some of the nodes if the coordinator hasn't shaken hands with someone; however,
+            // this existed largely as a defense against performance issues with concurrent schema modifications.
             versions = client.describe_schema_versions();
-            if (uniqueSchemaWithQuorumAgreementAndOtherNodesUnreachable(config, versions)) {
```

Comment:
we should delete the `CassandraKeyValueServiceConfig config` from this method's signature and remove its usage across the entirety fo the method

Reply:
It's still used for the blocking interval, though I've changed that to passing in an int.